_includes/charts/calico/templates/calico-kube-controllers.yaml

```diff
@@ -73,11 +73,20 @@ spec:
             # Mount in the etcd TLS secrets.
             - mountPath: /calico-secrets
               name: etcd-certs
+          livenessProbe:
+            exec:
+              command:
+              - /usr/bin/check-status
+              - -r
```

Comment:
Strictly speaking, the `-r` flag was intended to mean "ready", and we intended to add a "-l" flag to mean "alive" - code is here: https://github.com/projectcalico/kube-controllers/blob/master/cmd/check-status/main.go#L37

I think we can probably get away with this as a quick fix, but should we want to have separate ready / live conditions we'll need to have separate flags, so longer term I'd definitely like to see that even if for now `-l` == `-r`. 

Would you be interested in adding a new option?

Reply:
Ah, didn't know this. Would not mind adding a new flag for liveness in a separate PR.

Reply:
https://github.com/projectcalico/kube-controllers/pull/655